imports/plugins/core/shop/server/queries/test.js

```diff
@@ -0,0 +1,21 @@
+import queries from "./";
+
+const context = {
+  dataLoaders: {
+    Shops: {
+      load: jest.fn(() => [
+        {
+          _id: 1,
```

Comment:
`Shop._id` is a String in the schema `imports/collections/schemas/shops.js` so let's reflect that here for consistency. I think all of our DB _id properties are strings. There are a bunch of places in this PR using numbers. Let's change them all to strings.